integration/embedded_derp_test.go

```diff
@@ -239,10 +239,13 @@ func (s *EmbeddedDERPServerScenario) CreateHeadscaleEnv(
 
 		if clientCount.WebsocketDERP > 0 {
 			// Containers that use DERP-over-WebSocket
+			// Note that these clients *must* be built
+			// from source, which is currently
+			// only done for HEAD.
 			err = s.CreateTailscaleIsolatedNodesInUser(
 				hash,
 				userName,
-				"all",
+				"head",
```

Comment:
This can be the `VersionHead` constant you added

Reply:
Good idea, I probably missed this instance because I only looked at the linter output when I introduced that constant.